egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -305,7 +331,7 @@ def ParseSpliceString(splice_indexes):
     if (num_hidden_layers < args.num_lstm_layers):
         sys.exit("--num-lstm-layers : number of lstm layers has to be greater than number of layers, decided based on splice-indexes")
 
-    left_context = left_context + args.bptt_truncation_width + args.context_sensitive_chunk_width
+    left_context = left_context + args.chunk_left_context
```

Comment:
It had to be provided to the make_config.py script as the same script also dumps the variables necessary for get_egs.sh. I will make necessary changes.